arch/arm/src/tiva/Make.defs

```diff
@@ -53,6 +53,7 @@ CMN_CSRCS += up_releasepending.c up_releasestack.c up_reprioritizertr.c
 CMN_CSRCS += up_schedulesigaction.c up_sigdeliver.c up_stackframe.c
 CMN_CSRCS += up_svcall.c up_trigger_irq.c up_unblocktask.c up_udelay.c
 CMN_CSRCS += up_usestack.c up_vfork.c
+CMN_CSRCS += up_hostfs.c
```

Comment:
There is a check inside of up_hostfs.c, but it would speed compile times a tiny amount if this were enclosed in if ($(CONFIG_ARM_SEMIHOSTING_HOSTFS),y) so that the dependency is clearer and we don't wast the little bit of time compiling an empty file.

Reply:
it makes sense. i'll add the make conditional. and probably remove the guard in the C file.